### DISCUSSION TITLE
Problem installing on RPI5 -> create installable package?

### DISCUSSION THREAD

**Discussion by bromic1** [blueman-project/blueman#2434] [Q&A]
blueman: 2.4.2
BlueZ: 5.66
Distribution: Raspberry Pi OS 64bit
Desktop environment: Debian Bookworm

Hi @cschramm, could you pls also create an installable package for Python 3.11, arm64 for the package manager? 
I have the problem that I cannot get the latest version installed by myself. I added /usr/lib/python3.11/site-packages to PYTHONPATH and started the installation with --prefix=/usr.
I still get failures during starting blueman-sendto which says like timeouts during calling StartServiceByName for org.blueman.Applet... Blueman applet needs to be running...

In the included version 2.3.5 there is for some reason no (re)search button in blueman-sendto. So it does only search 1 cycle when I start the app. When the device is not found I would have to restart the app. 

**EDIT: I got it to work by starting the applet manually but the (re)search button is still not there... So maybe this is another issue?**

**Comment by cschramm** [comment]
Are you referring to https://blueman.cschramm.eu/debian/? I don't think I can provide arm64 there with reasonable effort as the packages get built on an x86_64 server. You can grab [Debian's 2.4.1-1](https://snapshot.debian.org/package/blueman/2.4.1-1/#blueman_2.4.1-1). There won't be 2.4.2 for Python 3.11, though, as Debian is on 3.12 now (2.4.1+b1 is the rebuild for that).

You can easily build the package yourself for your architecture and Python version e.g. obtaining it with `git clone https://salsa.debian.org/debian/blueman.git` and running `dpkg-buildpackage -us -uc -nc` inside it (`apt-get build-dep blueman` can help with the build dependencies if the distribution's package isn't too old).

**Reply by stephematician** [reply]
Thanks for this comment. Building the Debian package from <https://salsa.debian.org/debian/blueman.git> ended up being the easiest option for me. As far as I am aware, it wasn't an option for me to use https://blueman.cschramm.eu/debian/  as my system has Python 3.12 (and only 3.9, 3.10, and 3.11 were provided by that APT repo). I was also able to compile and install using the source, however the package approach was better at placing the installed files out of the box (compared to fiddling with site-packages etc).
